src/quote/Quote.module.css

```diff
@@ -0,0 +1,10 @@
+.quote {
+  margin: 2rem;
+}
+
+.quote p {
+  margin: 3rem auto;
+  font-size: 1.4rem;
+  color: black;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly move your `Quote.module.css` to the `css-modules` directory you create mentioned above. It is good practice to house all similar files in the same directory and import them where required. That way your project will be very easy to browse through. In addition, it will make it look more professional and stand out.